api.js

```diff
@@ -710,7 +716,14 @@ class Api {
       utils: this.utils,
       pathLib: this.pathLib,
       fsLib: this.fsLib,
-      state
+      errorFormatter: this._errorFormatter,
+      state,
+
+      // Pass all help options (styling) down to the Context constructor. For the most
+      // part, these options will be ignored, as the Context usually does not care about
+      // styling - but it will pick up individual options if appropriate (such as
+      // `styleUnexpectedError` if there is unexpected error).
+      ...this.helpOpts
```

Comment:
It seems a little odd/unnecessary/inefficient to me to pass _all_ `helpOpts` to the Context, but I guess it doesn't hurt anything. (No action needed.)

Reply:
I feel like, once you take the plunge to say "we sometimes require styling in `Api`", you should just pass it all.

I'd be totally comfortable including it nested if you felt that was better encapsulated -- i.e. instead of `...helpOpts` pass in `helpOpts` as a key.

Reply:
> I'd be totally comfortable including it nested if you felt that was better encapsulated -- i.e. instead of `...helpOpts` pass in `helpOpts` as a key.

My gut instinct tells me this would be better than how it currently is, though I can't objectively defend that.

If you're willing to make that change, I'd appreciate it.